src/styles/utils/__reset.scss

```diff
@@ -0,0 +1,11 @@
+body,
+a,
```

Comment:
but AFAIK tag a don't have margin/padding

Reply:
Oh, thank you